pennylane_lightning_gpu/src/util/MPIManager.hpp

```diff
@@ -0,0 +1,551 @@
+#pragma once
+
+#include <bit>
+#include <complex>
+#include <cstring>
+#include <cuda.h>
+#include <cuda_runtime.h>
+#include <mpi.h>
+#include <stdexcept>
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
```suggestion
#include <cstdio>
#include <cstdlib>
```

We should use the C++ headers rather than the C headers where possible.

Reply:
done!